src/Symfony/Component/PropertyAccess/Annotation/PropertyAccessor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor configuration annotation.
+ *
+ * @Annotation
+ * @Target({"PROPERTY"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class PropertyAccessor extends ConfigurationAnnotation
+{
+    protected $setter;
```

Comment:
Alternatively, they can be `public` properties. It avoid the need for getters and setters.